src/web/Fig.Web/Pages/Groups.razor.css

```
@@ -0,0 +1,651 @@
+    gap: 6px;
+    flex-wrap: wrap;
+}
+
+.source-setting-chip .client-name {
+    font-weight: 600;
+    color: rgba(255, 255, 255, 0.9);
+}
+
+.source-setting-chip .setting-name {
+    color: rgba(255, 255, 255, 0.72);
+    word-break: break-word;
```

Stylelint (17.6.0)

[error] Unexpected deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)